projects/vdk-plugins/vdk-kerberos-auth/requirements.txt

```diff
@@ -1,5 +1,5 @@
 docker-compose
-minikerberos
+minikerberos==0.2.20
```

Comment:
add comment on why we are pinning it please. And create github issue to check it (Link it in the comment)